modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -66,27 +66,34 @@ StressDivergenceRSphericalTensors::computeQpOffDiagJacobian(unsigned int jvar)
 }
 
 Real
+#if DEBUG
 StressDivergenceRSphericalTensors::calculateJacobian(unsigned int ivar, unsigned int jvar)
+#else
+StressDivergenceRSphericalTensors::calculateJacobian(unsigned int /*ivar*/, unsigned int /*jvar*/)
+#endif
 {
```

Comment:
```suggestion
StressDivergenceRSphericalTensors::calculateJacobian(unsigned int ivar, unsigned int jvar)
{
  libmesh_ignore(ivar);
  libmesh_ignore(jvar);
```

Reply:
Even better:

```
StressDivergenceRSphericalTensors::calculateJacobian(unsigned int libmesh_dbg_var(ivar), unsigned int libmesh_dbg_var(jvar))
```

Reply:
Yeah, that's the one I had in mind but could not find anymore!